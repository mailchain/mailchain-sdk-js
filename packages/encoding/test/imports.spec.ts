import path from 'path';
import { getImportsFromDir } from '../../../../collect-imports';
import pckg from '../package.json';

describe('check imports snapshot', () => {
	const imports = [
		...new Set(
			getImportsFromDir(path.resolve(`${__dirname}`, '..'), ['.ts', '.js']).filter((it) =>
				it.startsWith(pckg.name),
			),
		),
	].sort();
	test('compare imports', () => {
		expect(imports).toMatchSnapshot('imports');
	});
});
