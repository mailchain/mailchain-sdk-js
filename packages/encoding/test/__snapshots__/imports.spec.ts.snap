// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check imports snapshot compare imports: imports 1`] = `
[
  "@mailchain/encoding/base32",
  "@mailchain/encoding/base58",
  "@mailchain/encoding/base64",
  "@mailchain/encoding/consts",
  "@mailchain/encoding/encoding",
  "@mailchain/encoding/hex",
  "@mailchain/encoding/utf8",
]
`;
